thirdparty/download-thirdparty.sh

```diff
@@ -218,6 +218,11 @@ do
             echo "nothing has been done with ${!NAME}"
             continue
         fi
+
+        if [ `ls $TP_SOURCE_DIR/tmp_dir/ | wc -l` != "1" ]; then
```

Comment:
what's the expected output of the `ls` ? shall directly check the existence of target directory with `-d`

Reply:
Have adapted the way you suggested.